handlers/geolocation/geolocation.go

```diff
@@ -20,6 +22,14 @@ import (
 	"github.com/livepeer/go-api-client"
 )
 
+const (
+	streamSourceRetries       = 10
+	streamSourceRetryInterval = 15 * time.Second
```

Comment:
I think we should probably reduce this a bit to try to speed up the response as much as possible. Maybe 2-5s?